packages/jspsych/src/timeline/Trial.ts

```diff
@@ -185,6 +192,7 @@ export class Trial extends TimelineNode {
       ...result,
       trial_type: this.pluginInfo.name,
       trial_index: this.index,
+      version: this.pluginInfo["version"] ? null : this.pluginInfo["version"],
```

Comment:
Should we use `plugin_version` instead of `version`? I think it might be semantically clearer since `version` could refer to all kinds of things.

Reply:
I thought about this as well, I like the change. The only reason didn't include simplicity and I thought we could go back and specify if necessary. Changed!